setup.py

```diff
@@ -15,7 +15,11 @@
     "networkx==1.11",
     "tqdm",
     "gitpython",
-
+    "torch",
+    "keras",
+    "pandas==0.25.3",
+    "tensorflow==1.15",
```

Comment:
Is there another solution than requiring users to `pip install` all 3 (pytorch, keras, tensorflow)?